### DISCUSSION TITLE
可否出一个windows可执行文件的release？

### DISCUSSION THREAD

**Discussion by loks666** [lobehub/lobehub#593] [General]
### 🥰 需求描述 | Feature Description

因为在网站内常常手滑将聊天内容都失去了，所以希望能有一个客户端

### 🧐 解决方案 | Proposed Solution

希望可以将项目打成exe包，直接安装在本地

### 📝 补充信息 | Additional Information

msi和exe可执行程序都可以

[a comment by an automated account is not shown]

**Comment by mushan0x0** [comment]
<img width="1037" alt="image" src="https://github.com/lobehub/lobe-chat/assets/29084441/bf305252-9171-4fa8-b03b-8d4e4ea6012e">
这是 PWA 的，目前你可以用这个

**Comment by arvinxx** [comment]
> ### 🥰 需求描述 | Feature Description
> 因为在网站内常常手滑将聊天内容都失去了，所以希望能有一个客户端
> 
> ### 🧐 解决方案 | Proposed Solution
> 希望可以将项目打成exe包，直接安装在本地
> 
> ### 📝 补充信息 | Additional Information
> msi和exe可执行程序都可以

我们有做本地记录持久化的，消息不会丢。 如果是类似输入过程中的这种问题，可以跟进 #563

**Reply by kindmesdilabso** [reply]
现在 https://chat-preview.lobehub.com/welcome 已经无法在境内访问了,还是希望能够出一个可安装执行程序

[a comment by an automated account is not shown]
